app/component/departure/DepartureTime.js

```diff
@@ -96,3 +96,36 @@ DepartureTime.propTypes = {
 };
 
 export default DepartureTime;
+
+
+/**
+ * maps stoptime to data structure required by DepartureTime. This is copied
+ * from departure-list-container.
+ *
+ *  @param stoptime stoptime from graphql
+ *  @param pattern pattern from graphql
+ */
+const mapStopTime = (stoptime, pattern) => (
```

Comment:
Should we make DepartureListContainer use this, or put this into a new common higher order component?